### PR TITLE
Adding brand colors to CSSinJS exports

### PR DESCRIPTION
**Pull request by georgewrmarshall**
### **Description**

This PR adds the `brandColors` object to the exports of the design tokens package. It also includes unit tests to ensure the `brandColors` object matches the Figma design token values.

1. Added `brandColors` object to the design tokens exports.
2. Created unit tests for `brandColors` to ensure it matches the Figma design tokens values.

Fixes: #581 

### Usage

```
import { brandColors } from '@metamask/design-tokens';

<Icon name={IconName.Star} color={brandColors.yellow.yellow500} />
```

### Manual testing steps
1. Pull this branch
2. Run the unit tests and ensure all tests pass.
4. Ensure `brandColors` is exported from project index

### CODE REVIEW

File: src/js/brandColors/brandColors.test.ts [hunk 1]
```diff
@@ -0,0 +1,410 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { brandColors } from './brandColors';
+
+const designTokens = require('../../figma/tokens.json');
+
+describe('Brand Color', () => {
+  describe('White', () => {
+    it('js tokens for white000 matches figma tokens white000', () => {
```
Comment (georgewrmarshall) [inline src/js/brandColors/brandColors.test.ts:8]: These tests check the export color values match the hex values of the colors in Figma json

File: src/js/brandColors/brandColors.test.ts [hunk 2]
```diff
@@ -0,0 +1,410 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { brandColors } from './brandColors';
+
+const designTokens = require('../../figma/tokens.json');
+
+describe('Brand Color', () => {
```
[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply src/js/brandColors/brandColors.test.ts]: Fixed in https://github.com/MetaMask/design-tokens/pull/582/commits/a9038d2c9a5eec8b4324fafd82a4de15ee419511

File: src/js/brandColors/brandColors.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
+import { BrandColors } from './brandColors.types';
+
+export const brandColors: BrandColors = {
+  white: {
+    white000: '#ffffff',
```
Comment (brianacnguyen) [inline src/js/brandColors/brandColors.ts:5]: why are we doing `white000` and not `000`?

Comment (georgewrmarshall) [reply src/js/brandColors/brandColors.ts]: As per our conversation and weighing up the pros and cons updated here https://github.com/MetaMask/design-tokens/pull/582/commits/5fab2d55d41430c23663f0debc4d9f886700210b

File: src/js/brandColors/brandColors.test.ts [hunk 4]
```diff
@@ -0,0 +1,410 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { brandColors } from './brandColors';
+
+const designTokens = require('../../figma/tokens.json');
+
+describe('Brand Colors', () => {
+  describe('White', () => {
+    it('js tokens for white000 matches figma tokens white000', () => {
+      expect(brandColors.white['000']).toStrictEqual(
+        designTokens.global.brandColors.white.white000.value.toLowerCase(),
```
Comment (brianacnguyen) [inline src/js/brandColors/brandColors.test.ts:10]: you forgot to update your tests XD

Comment (georgewrmarshall) [reply src/js/brandColors/brandColors.test.ts:10]: This is the Figma token json

**Review by brianacnguyen — APPROVED** [review]

**Pull request merged**